packages/webpack-cli/__tests__/ZeroConfigGroup.test.js

```diff
@@ -42,4 +42,15 @@ describe('GroupHelper', function() {
         const result = group.run();
         expect(result.options.mode).toEqual('development');
     });
+
+    it('should handle the mode option [none]', () => {
+        const group = new ZeroConfigGroup([
+            {
+                mode: 'none',
+            },
+        ]);
+
+        const result = group.run();
+        expect(result.options.mode).toEqual('development');
+    });
```

Comment:
The assertion is not correct. It should be `"none"`

Reply:
but when we pass `--mode=none` , it outputs development build .

Reply:
Also this test is failling if I replace it with `"none"`

Reply:
Which means it's a bug and code should be reviewed. You should rely on your testing suite, not on the code.
That's the point the test. (TDD)

Reply:
![Screenshot from 2020-02-28 15-28-02](https://user-images.githubusercontent.com/46647141/75538645-f13cfe80-5a3e-11ea-8868-1f14ea488094.png)

Reply:
Please keep the code for test the same and change the code which you are testing to make it pass the tests.

Reply:
@ematipico Yes there is a bug `--mode` is accepting any value and outputs development build. I can pass `--mode=abcd` , and it produces the build.

![Screenshot from 2020-02-28 15-33-34](https://user-images.githubusercontent.com/46647141/75539120-bedfd100-5a3f-11ea-9a7d-4c462ca2436c.png)

It should accept only `production , development or none`